homeassistant/components/tractive/sensor.py

```diff
@@ -82,7 +88,12 @@ def __init__(
     @callback
     def handle_status_update(self, event: dict[str, Any]) -> None:
         """Handle status update."""
-        self._attr_native_value = event[self.entity_description.key]
+        if self.entity_description.value_fn:
+            value = self.entity_description.value_fn(event[self.entity_description.key])
```

Comment:
Can't we add `value_fn` to the others too so we can skip the if-statement here?

Reply:
Good point, thanks